lang/python/python-sentry-sdk/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2015-2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-sentry-sdk
+PKG_VERSION:=0.9.0
```

Comment:
[0.9.5](https://pypi.org/project/sentry-sdk/#history) is available.

Reply:
Updated.